stackslib/src/config/mod.rs

```diff
@@ -2122,82 +2122,374 @@ impl NodeConfig {
     pub wait_on_interim_blocks: Option<Duration>,
     /// minimum number of transactions that must be in a block if we're going to replace a pending
     /// block-commit with a new block-commit
+    ///
+    /// Default: `0`
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
     pub min_tx_count: u64,
-    /// Only allow a block's tx count to increase across RBFs.
+    /// If true, requires subsequent mining attempts for the same block height
+    /// to have a transaction count >= the previous best attempt.
+    ///
+    /// Default: `false`
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
     pub only_increase_tx_count: bool,
-    /// Path to a script that prints out all unconfirmed block-commits for a list of addresses
+    /// Optional path to an external helper script for fetching unconfirmed block-commits.
+    /// Used to inform the miner's dynamic burn fee bidding strategy with off-chain data.
+    ///
+    /// If a path is provided, the target script must:
+    /// - Be executable by the user running the Stacks node process.
+    /// - Accept a list of active miner burnchain addresses as command-line arguments.
+    /// - On successful execution, print a JSON array representing `Vec<UnconfirmedBlockCommit>`
+    ///   (see [`stacks::config::chain_data::UnconfirmedBlockCommit`] struct) to stdout.
+    /// - Exit with code 0 on success.
+    ///
+    /// Look at [`test_get_unconfirmed_commits`](stackslib/src/config/chain_data.rs) for an example script.
+    ///
+    /// Default: `None` (feature disabled).
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode and
+    /// by the "get-spend-amount" cli subcommand.
     pub unconfirmed_commits_helper: Option<String>,
-    /// Targeted win probability for this miner.  Used to deduce when to stop trying to mine.
+    /// The minimum win probability this miner aims to achieve in block sortitions.
+    ///
+    /// This target is used to detect prolonged periods of underperformance. If the miner's
+    /// calculated win probability consistently falls below this value for a duration specified
+    /// by [`MinerConfig::underperform_stop_threshold`] (after an initial startup phase), the miner may
+    /// cease spending in subsequent sortitions (returning a burn fee cap of 0) to conserve resources.
+    ///
+    /// Setting this value close to 0.0 effectively disables the underperformance check.
+    ///
+    /// Default: `0.0`
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
     pub target_win_probability: f64,
     /// Path to a serialized RegisteredKey struct, which points to an already-registered VRF key
     /// (so we don't have to go make a new one)
+    ///
+    /// Default: `None`
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
```

Comment:
I don't think this is deprecated: https://github.com/stacks-network/stacks-core/blob/develop/testnet/stacks-node/src/nakamoto_node.rs#L366-L372